tests/plugins/pastefromlibreoffice/safarisupport.js

```diff
@@ -0,0 +1,29 @@
+/* bender-tags: editor,clipboard */
+/* bender-ckeditor-plugins: pastefromlibreoffice */
+
+( function() {
+	'use strict';
+
+	bender.editor = true;
+
+	bender.test( {
+		// (#3824)
+		'test handler is not loaded in Safari': function() {
```

Comment:
Could we cover all browsers with such test (or maybe at least those were the filter is turned off, since for others we have more detailed paste tests)?